src/excmds.ts

```diff
@@ -467,6 +469,26 @@ export function focusinput(nth: number|string) {
         // we can look up in future (tabindex, selector?), perhaps along with
         // a way to remember the page it was on?
     }
+    else if (nth === "-n") {
+        // attempt to find next input
+        let inputs = DOM.getElemsBySelector(INPUTTAGS_selectors,
+            [DOM.isSubstantial]) as HTMLElement[]
+        if (inputs.length) {
+            let index = inputs.indexOf(LAST_USED_INPUT) + 1
```

Comment:
What happens if LAST_USED_INPUT is null?

Reply:
If LAST_USED_INPUT is null, both `focusinput -n` and `focusinput -N` will focus first input. I think that's fine.